tests/timeseries_util.erl

```diff
@@ -41,101 +59,91 @@
 -define(MAXFLOAT,            math:pow(2, 63)).
 -define(MULTIPLECLUSTERSIZE, 3).
 
-confirm_create(ClusterType, DDL, Expected) ->
+-spec(confirm_create(atom(), string()) -> {ok, string()} | term()).
+confirm_create(ClusterType, DDL) ->
 
-    [Node | _] =build_cluster(ClusterType),
+    [Node | _] = build_cluster(ClusterType),
 
     Props = io_lib:format("{\\\"props\\\": {\\\"n_val\\\": 3, \\\"table_def\\\": \\\"~s\\\"}}", [DDL]),
-    Got = rt:admin(Node, ["bucket-type", "create", get_bucket(), lists:flatten(Props)]),
-    ?assertEqual(Expected, Got),
-
-    pass.
+    rt:admin(Node, ["bucket-type", "create", get_bucket(), lists:flatten(Props)]).
 
-confirm_activate(ClusterType, DDL, Expected) ->
+confirm_activate(ClusterType, DDL) ->
 
     [Node | Rest] = build_cluster(ClusterType),
     ok = maybe_stop_a_node(ClusterType, Rest),
     {ok, _} = create_bucket(Node, DDL, 3),
-    Got     = activate_bucket(Node, DDL),
-    ?assertEqual(Expected, Got),
-
-    pass.
+    activate_bucket(Node, DDL).
 
-confirm_put(ClusterType, TestType, DDL, Obj, Expected) ->
+confirm_put(ClusterType, TestType, DDL, Obj) ->
```

Comment:
Yes, I'm not crazy about the organization.  I'm guessing the original author just wanted to minimize boilerplate code which is why the asserts were in there in the first place.